bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java

```diff
@@ -103,7 +108,7 @@ private void poll() throws FreeboxException {
             updateChannelQuantity(SYS_INFO, UPTIME, uptime, Units.SECOND);
         } else {
             logger.debug("{}: poll with reachable={}", thing.getUID(), reachable);
-            updateStatus(ThingStatus.OFFLINE);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE, "Player is not reachable");
```

Comment:
I see there is already I18N support for status descriptions in the binding:
https://github.com/openhab/openhab-addons/blob/1f17e53bc2eb277b81c8da08e35aef58a928d3af/bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/FreeboxOsHandler.java#L74-L75

so maybe this one should support I18N as well?

Reply:
Yes, done.